newIDE/app/src/MainFrame/index.js

```diff
@@ -1534,6 +1566,21 @@ class MainFrame extends React.Component<Props, State> {
     });
   };
 
+  _onSetPreview = (editorTab: number) => {
+    const { currentProject } = this.state;
+    //Get name of scene on tab clicked
+    let name = editorTab.editorRef.props.layoutName;
+    this.setState(
+      {
+        scenePreviewOverride: currentProject.getLayout(name),
```

Comment:
Maybe you can continue without fixing this for now, but I would prefer to store the scene to be used for preview as a string. You can store this string in the state as `previewFirstSceneName`.

In general, it's risky to store a GDevelop.js object like this: 
* If I use this scene as preview override, and then delete it in the project manager, there is still a reference to it here => crash when you try to launch the preview.

Instead prefer to store the name of the layout, and find it at the moment of the preview (and if you don't find it, just ignore it - must have been deleted or it's a wrong name for some reason).